cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,19 @@
 /// <reference types="cypress" />
 
+const { generateNewUser } = require("../support/generateNewUser");
+
 describe('Sign In page', () => {
+
   it('should provide an ability to log in', () => {
+    cy.signUp().then(({ username, email, password }) => {
+      cy.visit('https://react-redux.realworld.io/#/login?_k=qt0jum')
```

Comment:
You have baseUrl in config - should work like this.
```suggestion
      cy.visit('/#/login?_k=qt0jum')
```